docs/guides/installation-tce.md

```diff
@@ -0,0 +1,55 @@
+# TriggerMesh Installation for Tanzu Community Edition
+
+[Tanzu Community Edition](https://tanzucommunityedition.io/) is a free and open source Kubernetes platform provided by VMware.
+
+!!! Info "Tanzu Community Edition package repository versions"
+       The version of `knative-serving` currently in the Tanzu Community Edition package repository is version 0.22 which does not meet TriggerMesh's prerequisites. There is also currently not a `knative-eventing` package available, so please follow the instructions below.
+
+## Installation
+
+1. Follow the [Getting Started with Tanzu Community Edition](https://tanzucommunityedition.io/docs/latest/getting-started-standalone/) installation instructions and create your Tanzu cluster on your platform of choice.
+1. Ensure you are in the context of your Tanzu cluster
+```
+kubectl config use-context <STANDALONE-CLUSTER-NAME>-admin@<STANDALONE-CLUSTER-NAME>
+```
+1. Install Knative Serving following the [Installing Knative Serving using YAML files](https://knative.dev/docs/admin/install/serving/install-serving-with-yaml/) instructions.
+```
+kubectl apply -f https://github.com/knative/serving/releases/download/v0.26.0/serving-crds.yam
+kubectl apply -f https://github.com/knative/serving/releases/download/v0.26.0/serving-core.yaml
+```
+1. Wait until the pods have started before proceeding.
+```
+kubectl get pods -n knative-serving
```

Comment:
It would be useful if a sample output is also listed, so that the reader can see how many pods are created and expect the same at their end

Reply:
I'll put that in the blog post I guess.